packages/attention/index.js

```diff
@@ -1,76 +1,163 @@
 import { css, html } from 'lit';
-import { FabricElement, fclasses } from '../utils';
+import { FabricElement, classes } from '../utils';
+import { attention as c } from '@fabric-ds/css/component-classes';
+import { opposites, rotation, useRecompute as recompute } from '@fabric-ds/core/attention';
 
 class FabricAttention extends FabricElement {
+  static properties = {
+    // Render Attention element without arrow
+    noArrow: { type: Boolean, reflect: true },
+    // Whether Attention element is shown, used for tooltip
+    isShowing: { type: Boolean, reflect: true },
+    // Placement according to the target element
+    // Arrow would be on the opposite side of this position
+    placement: { type: String },
+    // Id of element the Attention component is rendered relatively to
+    targetElId: { type: String },
+
+    // Arrow props
+    tooltip: { type: Boolean, reflect: true },
+    callout: { type: Boolean, reflect: true },
+    popover: { type: Boolean, reflect: true },
+  };
+
   static styles = css`
     :host {
       display: block;
     }
+
+    #arrow {
+      border-top-left-radius: 4px;
+      z-index: 1;
+    }
+
+    ::slotted(:last-child) {
+      margin-bottom: 0px !important;
+    }
   `;
 
-  static properties = {
-    actualDirection: { type: String, state: true },
+  connectedCallback() {
+    super.connectedCallback();
 
-    // Render Attention element without arrow
-    noArrow: { type: Boolean },
+    if (!this.placement || !Object.keys(opposites).includes(this.placement)) {
+      throw new Error(
+        `Invalid "placement" attribute. Set its value to one of the following:\n${JSON.stringify(
+          Object.keys(opposites),
+        )}`,
+      );
+    }
 
-    // Whether Attention element is shown, used for tooltip
-    isShowing: { type: Boolean },
+    /*     // Attention of "callout" type is used inline
+    if (this._isCallout) {
+      this.style.position = 'relative';
+    } */
+  }
 
-    // Placement according to the target element
-    // Arrow would be on the opposite side of this position
-    placement: { type: String },
+  get _isShowing() {
+    return this.isShowing;
+  }
```

Comment:
Why do we map isShowing to _isShowing here?

Reply:
Actually it's unnecessary, thanks! Removed redundant getters in [c55e82b](https://github.com/fabric-ds/elements/pull/71/commits/c55e82b260c658386dbd68a569dc977637236f50)